roles/tripleo_cluster_set_container_cli/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# defaults file for tripleo_ceph_set_container_cli
+ceph_dot_conf: "{{ ceph_config_home}}/ceph.conf"
+ceph_keyring: "{{ ceph_config_home }}/ceph.client.admin.keyring"
+ceph_container_ns: "docker.io/ceph"
+ceph_container_image: "ceph"
+ceph_container_tag: "v15"
+container_cli: "podman"
+container_options: "--net=host --ipc=host"
```

Comment:
Nice. Seb was talking about ipc=host on the video: https://www.youtube.com/watch?v=MaKxsrZStDc